usaspending_api/etl/migrations/0004_create_transaction_id_seq.py

```diff
@@ -0,0 +1,21 @@
+"""
+Create the "transaction_id_seq" as a Postgres sequence used to keep track of ID values in
+"transaction_id_lookup" for Delta.
+"""
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("etl", "0003_duns_schema_change"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql="CREATE SEQUENCE IF NOT EXISTS transaction_id_seq AS BIGINT",
+            reverse_sql="DROP SEQUENCE IF EXISTS transaction_id_seq"
```

Comment:
Do we need to "seed" the sequence with the value it starts from or is that handled in the code?